main.cpp

```diff
@@ -229,12 +216,8 @@ bool Computer::playMusic(int howLoud)
     {
         return true;
     }
-    else
-    {
         loudness = 0; //there is only one loudness and that's 11 or turn it off!!!!
         return false;
```

Comment:
Just because you delete the `else { }` doesn't mean you can ignore the formatting and not fix the indentation.
- [ ] revise